### PR TITLE
OG-4095 Remove bootstrap dependency

### PR DESCRIPTION
**Pull request by nabaroa**
## 🖼 Context

📎 https://transferwise.atlassian.net/browse/OG-4095

When the monorepo was created it had bootstrap as dependency.

## 🚀 Changes
This PR gets rid of it.
Changes:

- Removed boostrap as dependency
- Add proper font loader to nextjs webpack conf. Similar context in: https://github.com/transferwise/neptune-web/pull/620
- Add CSS components (jumbotron/pagination/circle backgrounds) code from Bootstrap and adapt to Neptune custom props and variables.
- Improvements of jumbotron and circleBackground documentation.

## ✅ Checklist

- [ ] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/marketing-components/blob/main/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/marketing-components/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary
- [ ] Change has been approved by someone outside of your team

### CODE REVIEW

File: packages/docs/pages/css/Jumbotron.mdx [hunk 1]
```diff
@@ -75,14 +74,14 @@ To provide a background image for a jumbotron unit, wrap the unit with <code> ju
...
-      <a className="btn btn-primary btn-lg" href="#" role="button">
+      <a className="btn btn-primary" href="#" role="button">
         Learn more
       </a>
     </p>
   </div>
 </div>
 
-    <div class="jumbotron-image-inverse" style="background-image: url('https://daurzqvz85pz.cloudfront.net/55fabdde87a2a-testimonial_naila_background.png')">
+    <div class="jumbotron-image-inverse" style="background-image: url('...')">
```
Comment (danlaush) [inline packages/docs/pages/css/Jumbotron.mdx:84]: Intentional?

Comment (nabaroa) [reply packages/docs/pages/css/Jumbotron.mdx:84]: Yes, that's the demo part

imagen

File: packages/marketing-components/src/videomodal/VideoModal.js [hunk 2]
```diff
@@ -60,8 +60,8 @@ VideoModalBody.propTypes = {
   onDismiss: Types.func.isRequired,
   isVisible: Types.bool.isRequired,
   style: Types.shape({
-    opacity: Types.any,
-    scale: Types.any,
+    opacity: Types.object,
```
Comment (danlaush) [inline packages/marketing-components/src/videomodal/VideoModal.js]: I think Richard's points are good - can make `opacity: Types.number` and `scale: Types.string`

File: packages/docs/pages/css/Jumbotron.mdx (hunk 1 above)
Comment (danlaush) [reply packages/docs/pages/css/Jumbotron.mdx:84]: duhhhhh sorry sorry, I didnt' see it was in an `mdx` file 🙈

File: packages/marketing-components/src/videomodal/VideoModal.js (hunk 2 above)
Comment (nabaroa) [reply packages/marketing-components/src/videomodal/VideoModal.js]: Yes, I know, but they were failing as well. The console said they were object, so I tried, but failed. I added another fix disabling the linter.

**Review by danlaush — APPROVED** [review]

**Pull request merged**